src/System.Windows.Forms/src/System/Windows/Forms/Control.cs

```diff
@@ -10853,6 +10991,17 @@ private protected void SetTopLevelInternal(bool value)
         }
     }
 
+    private static unsafe void PrepareDarkMode(HWND hwnd, bool darkModeEnabled)
```

Comment:
```suggestion
    private static unsafe void PrepareDarkMode(HWND hwnd, BOOL darkModeEnabled)
```

...and perfrom the cast at the callsite. This makes the intent obvious.

Reply:
Not sold on the method name either, it doesn't feel like it belongs to the vocabulary of this codebase (i.e., it's not very customary to call methods "PerpareXyz" here). It's probably more appropriate to call this `SetDarkMode`.

Reply:
It's not a public method?

Reply:
It is part of the codebase, and it needs to be maintained. The more cohesive the codebase is, the less mental tax is.

Reply:
I'll take a look.

Reply:
Would `InitializeDarkMode` more appropriate in your opinion?

Reply:
I've got to admit it's been a while since I look at the codebase, but `Initialize` feels more "native" to the codebase.
Thanks for asking.

Reply:
You got it!